zio-http/src/main/scala/zio/http/middleware/RequestLogging.scala

```diff
@@ -55,8 +55,8 @@ private[zio] trait RequestLogging { self: RequestHandlerMiddlewares =>
                   LogAnnotation(name.toString, value.toString)
               }.toSet
 
-            val requestBody  = if (request.body.isComplete) request.body.asChunk.map(Some(_)) else ZIO.none
-            val responseBody = if (response.body.isComplete) response.body.asChunk.map(Some(_)) else ZIO.none
+            val requestBody  = if (request.body.isComplete) request.body.asChunk.option else ZIO.none
+            val responseBody = if (response.body.isComplete) response.body.asChunk.option else ZIO.none
```

Comment:
Maybe we could log any errors happening here instead of just ignoring them

Reply:
I thought the error will occur in the userland again and should be handled there. What do you think?